terraform/shared/modules/gha-oidc-role/main.tf

```diff
@@ -46,4 +48,11 @@ variable "repo" {
 variable "branch" {
   type        = string
   description = "The branch of the repository allowed to assume the role"
+  default     = null
+}
+
+variable "environment" {
+  type        = string
+  description = "The GitHub environment allowed to assume the role"
+  default     = null
 }
```

Comment:
:nail_care: It's idiomatic to put `variable` declarations in its own file `variables.tf`

Reply:
PR welcome 😁

Reply:
:pray:  https://github.com/rust-lang/simpleinfra/pull/671